### PR TITLE
- Show calculated norm value for both passed and failed case

### PR DESCRIPTION
**Pull request by yeonsily**
- Beautify the test result
- Return exit code

### CODE REVIEW

**Comment by yeonsily** [comment]
I have no idea why CI check is failing. Can someone check the CI build log and let me know if this is a real issue?

**Comment by sayantan-nervana** [comment]
Yes, CI looks good. Will investigate why it fails

**Comment by yeonsily** [comment]
Was using python3 to run format check script.

File: diagnostics/model_test/verify_model.py [hunk 1]
```diff
@@ -211,6 +212,12 @@ def parse_json():
 
     assert all(
         [i == j for i, j in zip(out_tensor_names_cpu, out_tensor_names_ngraph)])
+    passed = True
+    th_dict = {
```
Comment (shresthamalik) [inline diagnostics/model_test/verify_model.py]: Please rename th_dict to threshold_dict.

**Pull request merged**